packages/minimongo/cursor.js

```diff
@@ -44,17 +44,24 @@ export default class Cursor {
    * @summary Returns the number of documents that match a query.
    * @memberOf Mongo.Cursor
    * @method  count
+   * @param {boolean} [applySkipLimit=false] If set to `true`, the value
+   *                                         returned will be limited to the
+   *                                         value supplied for `limit`, if one
+   *                                         was provided.
    * @instance
    * @locus Anywhere
    * @returns {Number}
    */
-  count() {
+  count(applySkipLimit = false) {
     if (this.reactive) {
       // allow the observe to be unordered
       this._depend({added: true, removed: true}, true);
     }
 
-    return this._getRawObjects({ordered: true}).length;
+    return this._getRawObjects({
+      ordered: true,
+      ignoreSkipLimit: !applySkipLimit
```

Comment:
I think it might make sense to just carry the `applySkipLimit` variable naming throughout, instead of switching (and inverting) it to `ignoreSkipLimit`. This would help simplify the code a bit and makes the logic easier to follow. Also, since `applySkipLimit` is a Mongo `count()` supported option, most people reviewing the code will already be familiar with it.

Reply:
Agreed! I'll get it updated